### DISCUSSION TITLE
Gridmaps : improve the active nodes display on gridmaps

### DISCUSSION THREAD

**Discussion by cyprien-griot** [IPGP/webobs#145] [Ideas]
Greetings all,

We thought about an improvment opportunity that could be made on gridmaps : indeed, inactive nodes are plotted after active ones, which makes those ones less visible. It would be better to focus on active nodes and make them more visible.

François suggested either implementing an option or to force active nodes plotting to be done after inactive ones, or to plot nodes in chronological order of station setup.

Regards,
Cyprien

**Comment by cyprien-griot** [comment]
Greetings all,
I'm taking the liberty of reviving this topic as our users are still bothered by this issue.
Regards,
Cyprien

**Comment by beaudu** [comment]
Done with commit 1856784aa6fcda1bff6e2b9f20fc4e5ed73d6046. Will be available in the next release.
